.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,232 @@
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+
+env:
+  fail-fast: true
+
+jobs:
+  phpunit-smoke-check:
+    name: "PHPUnit with SQLite"
+    runs-on: "ubuntu-20.04"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.2"
+          - "7.3"
+          - "7.4"
+        deps:
+          - "fixed"
```

Comment:
Should we use `"normal"` here? You are using `composer update`, so the dependencies are not fixed.